packages/mongodb-dialects/java-driver/src/main/kotlin/com/mongodb/jbplugin/dialects/javadriver/glossary/JavaDriverDialectParser.kt

```diff
@@ -612,7 +613,7 @@ object JavaDriverDialectParser : DialectParser<PsiElement> {
         val valueExpr = expression.argumentList.expressions.getOrNull(1) ?: return null
 
         val fieldName = keyExpr.tryToResolveAsConstantString()
-        val accumulatorExpr = parseComputedExpression(valueExpr, createsNewField = false)
+        val accumulatorExpr = parseComputedExpression(valueExpr)
```

Comment:
Realised that for the _id field, the underlying string will always point to a field from schema so I switched createsNewField to false and it turns out that then it is false always, so I removed it entirely.

Reply:
Actually, realised that _id can also be null so that needs to be accounted for. I added another [commit](https://github.com/mongodb/intellij/pull/101/commits/d39254509f882f8242fba6eefc884aad51d8d2ad) to address that.